test/NameBinding/scope_map_top_level.swift

```diff
@@ -21,7 +21,7 @@ extension Int {
 var i: Int = b.my_identity()
 
 
-// RUN: %target-swift-frontend -dump-scope-maps expanded %s 2> %t.expanded
+// RUN: %target-swift-frontend -enable-astscope-lookup -dump-scope-maps expanded %s 2> %t.expanded
```

Comment:
Might be nice to add a comment explaining why. I'm puzzled--why *does* it fail without the extra flag?

Reply:
If that would be an undue burden, you can make a radar and assign it to me.

Reply:
This is actually not needed for this change. When I checked with `EnableASTScope = false`, it just hit an assertion because  `-dump-scope-maps` requires ASTScope enabled.

I don't think it's a big deal because `-dump-scope-maps` is just a debug action. Or do you want to gracefully emit an error and exit? If so, I can remove this change and file a radar for you.

Reply:
How about leaving it in and adding a comment: "-dump-scope-maps requires ASTScopes" ?